inspirehep/modules/workflows/utils/__init__.py

```diff
@@ -269,3 +271,39 @@ def insert_wf_record_source(json, record_uuid, source):
     else:
         record_source.json = json
     db.session.commit()
+
+
+def resolve_validation_callback_url():
+    """Resolve validation callback.
+
+    Returns the callback url for resolving the validation errors.
+
+    Note:
+        It's using ``inspire_workflows.callback_resolve_validation``
+        route.
+    """
+    return url_for(
+        'inspire_workflows.callback_resolve_validation',
+        _external=True
+    )
+
+
+def validation_exception_to_validation_errors(data, schema):
+    """Creates a ``validation_errors`` dictionary from validation exception.
+
+    Args:
+        data (dict): the object to validate.
+        schema (str): the name of the schema.
+
+    Returns:
+        dict: ``validation_errors`` formatted dict.
+    """
+    validation_exceptions = validation_errors(data, schema=schema)
```

Comment:
The naming here is quite confusing :/

Reply:
hmm indeed, we made a change on the ``inspire-schema`` part